chia/wallet/puzzles/did_innerpuz.clvm

```diff
@@ -50,36 +50,38 @@
     (list ASSERT_COIN_ANNOUNCEMENT (sha256 (sha256 coin_id (sha256tree1 (make_message_puzzle my_id new_innerpuz pubkey))) my_id))
   )
 
-(defun-inline create_coin_ID_for_recovery (MOD_HASH LAUNCHER_ID LAUNCHER_PUZZLE_HASH parent innerpuzhash amount)
-    (sha256 parent (calculate_full_puzzle_hash MOD_HASH LAUNCHER_ID LAUNCHER_PUZZLE_HASH innerpuzhash) amount)
+  (defun create_coin_ID_for_recovery (SINGLETON_STRUCT launcher_id parent innerpuzhash amount)
```

Comment:
I find constructs like in line 46 to be a lot more legible using `list`. For example,

`(list (list CREATE_COIN_ANNOUNCEMENTS recovering_coin) (AGG_SIG_UNSAFE pubkey newpuz))`